orttraining/orttraining/core/framework/module_gradient_graph_builder.cc

```diff
@@ -228,19 +228,19 @@ void ModuleGradientGraphBuilder::HandleOutputsAndGrads() {
     attributes.insert({non_differentiable_outputs_name, non_differentiable_outputs});
   }
 
-  // YieldOps full_shape_outputs attribute specifies the indices of outputs that must be full shape.
+  // BreakOps full_shape_outputs attribute specifies the indices of outputs that must be full shape.
   // We need this info to set make TypeAndShapeInferenceFunction work properly.
   ONNX_NAMESPACE::AttributeProto full_shape_outputs;
   const std::string full_shape_outputs_name = "full_shape_outputs";
   full_shape_outputs.set_name(full_shape_outputs_name);
   full_shape_outputs.set_type(ONNX_NAMESPACE::AttributeProto::INTS);
 
-  std::vector<NodeArg*> yield_input_node_args;
-  std::vector<NodeArg*> yield_output_node_args;
+  std::vector<NodeArg*> stop_input_node_args;
```

Comment:
>stop_input_node_args [](start = 24, length = 20)

Have we agree on the name yet? Yield/Stop/Break

Reply:
I commented on this elsewhere. Yield is a more precise name than Stop or Break for what we're doing here.

Reply:
@mrry I also commented with my rationale why it is _not_ a more precise name. I also pinged you offline and indicated we will have an offline sync on the naming, so please do not block on this and continue with the remainder of the review. Thank you for taking the time to review my code.